### DISCUSSION TITLE
FontConfig config file not found (null)

### DISCUSSION THREAD

**Discussion by Raizencorp** [fonttools/fonttools#3188] [Q&A]
Hi everyone,

I'm curently working on a project with pyinstaller and weasyprint.

And as for the Title i'm having an issue with the config file of the fonts. The thing is i'm feeding pyinstaller with this file config [here](https://github.com/tschoonj/GTK-for-Windows-Runtime-Environment-Installer/tree/master/gtk-nsis-pack/etc/fonts) but Fontconfig doesn't manage to get them. and i follow the same dir names as linux and gtk etc/fonts/<files>

So I'd like to understand how is he looking for the conf file.

thanks for your help :)

**Comment by justvanrossum** [comment]
This may be the wrong project to ask. FontTools has no direct connection to FontConfig, nor to PyInstaller.

**Reply by Raizencorp** [reply]
oh i though fonttools and fontconfig were realated, mybad
